이거.js

```diff
@@ -0,0 +1,119 @@
+let today = new Date();
+
+//오늘의 현재 요일 표기
+let dayweek = today.getDay();
+let week = new Array(
+    "SUN",
+    'MON',
+    "TUE",
+    "WED",
+    "THU",
+    "FRI",
+    "SAT");
+let day_of_week = week[dayweek]
```

Comment:
자바스크립트에서 변수나 함수명을 작성할때는 카멜 케이스(lowerCamelCase)를 사용합니다. 각 단어의 첫 문자를 대문자로 표시하되, 이름의 첫 문자는 소문자로 적는 방법입니다.

ex) camelCase, backgroundColor, className

```suggestion
const dayOfWeek = week[dayweek];
```

추가적으로 세미콜론 꼭 붙여주세요. 이하 같은 부분에서 동일 코멘트는 생략하겠습니다.